force-app/main/default/classes/ContactTriggerHandler.cls

```diff
@@ -0,0 +1,46 @@
+public with sharing class ContactTriggerHandler extends TriggerHandler{
+    private static Boolean isTriggerExecuted = false;
+    
+    public override void afterUpdate() {
+        if(isTriggerExecuted) {
+            return;
+        }
+        isTriggerExecuted = true;
+        List<Contact> contactsToUpdate = new List<Contact>();
+        for (Contact cont : (List<Contact>)Trigger.new) {
+            if(cont.DummyJSON_Id__c != null && Integer.valueOf(cont.DummyJSON_Id__c) > 100) {
+                Contact con = new Contact(Id = cont.Id);
+                DummyJSONCallout.postCreateDummyJSONUser(cont.Id);
```

Comment:
In a real scenario, you probably would not want to do a callout in a loop. You are limited to 50 @future methods per transaction. Is there ways you can think of bulkify this if you had more than 50 records changed?